multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/compose/elements/MarkdownTable.kt

```diff
@@ -63,8 +61,8 @@ fun MarkdownTable(
     val tableCornerSize = LocalMarkdownDimens.current.tableCornerSize
 
     val columnsCount =
-        remember { node.findChildOfType(HEADER)?.children?.count { it.type == CELL } ?: 0 }
-    val tableWidth by derivedStateOf { columnsCount * tableCellWidth }
+        remember(node) { node.findChildOfType(HEADER)?.children?.count { it.type == CELL } ?: 0 }
+    val tableWidth = remember(columnsCount, tableCellWidth) { columnsCount * tableCellWidth }
```

Comment:
I suppose theoretically not even the remember is needed here as the multiplication is trivial